app/models/metric_rollup.rb

```diff
@@ -1,8 +1,8 @@
 class MetricRollup < ApplicationRecord
   include Metric::Common
 
-  def self.find_all_by_interval_and_time_range(interval, start_time, end_time)
-    where(:capture_interval_name => interval, :timestamp => start_time..end_time)
+  def self.with_interval_and_timestamp(interval, timestamp)
```

Comment:
sorry for being picky... why the method name change?  aren't you still passing it a time range object?